poonam/JS/Assignment6_js/string.html

```diff
@@ -0,0 +1,149 @@
+                var output5 = str5.slice(ind1, ind2);
+                output5 = document.getElementById("res5").innerHTML= "Output = "+output5;
+            }
+            //lower case
+            function lower_case(){
+                var str6 = document.getElementById("val7").value;
+                var output6 = str6.toLowerCase();
+                output6 = document.getElementById("res6").innerHTML= "Output = "+output6;
+            }
+            //upper case
+            function upper_case(){
+                var str7 = document.getElementById("val8").value;
+                var output7 = str7.toUpperCase();
+                output7 = document.getElementById("res7").innerHTML= "Output = "+output7;
+            }
+            //search
+            function search_str(){
+                var str8 = document.getElementById("val9").value;
+                var word = document.getElementById("word").value;
+                var output8 = str8.search(word);
+                output8 = document.getElementById("res8").innerHTML= "Output = "+output8;
+            }
+            //replace
+            function replace_str(){
+                var str9 = document.getElementById("val10").value;
+                var nword = document.getElementById("nword").value;
+                var rword = document.getElementById("rword").value;
+                var output9 = str9.replace(nword, rword);
+                output9 = document.getElementById("res9").innerHTML= "Output = "+output9;
+            }
+        </script>
+    </head>
+    <body>
+        <form action="#">
+            <h3>Trim string</h3>
+            <label for="val">Enter text:</label>
+            <input type="text" name="val" id="val"><br>
+            <button type="button" onclick="trim()">Submit</button>
+            <h4 id="res"></h4>
+            <h3>String length</h3>
+            <label for="val1">Enter text:</label>
+            <input type="text" name="val1" id="val1"><br>
+            <button type="button" onclick="strlength()">Submit</button>
+            <h4 id="res1"></h4>
+            <h3>Index</h3>
+            <label for="val2">Enter text:</label>
+            <input type="text" name="val2" id="val2"><br><br>
+            <label for="ind1">Find index of:</label>
+            <input type="text" id="ind1"><br>
+            <button type="button" onclick="index()">Submit</button>
+            <h4 id="res2"></h4>
+            <h3>Last index</h3>
+            <label for="val3">Enter text:</label>
+            <input type="text" name="val3" id="val3"><br><br>
+            <label for="ind2">Find index of:</label>
+            <input type="text" id="ind2"><br>
+            <button type="button" onclick="lastindex()">Submit</button>
+            <h4 id="res3"></h4>
+            <h3>Concat</h3>
+            <label for="val4">Enter text:</label>
+            <input type="text" name="val4" id="val4"><br><br>
+            <label for="val5">First index:</label>
```

Comment:
label should be different here as its for concat strings like enter second textso give appropriate labels everywhere.